tests/integration/test_method_equivalence.py

```diff
@@ -208,13 +189,9 @@ def test_equivalence(approximate_model):
     subject to some optimization).
     """
     gpr_model = _create_full_gp_model()
-    gpr_likelihood = -gpr_model.log_likelihood()
-    if isinstance(approximate_model, gpflow.models.SVGP):
-        approximate_likelihood = -approximate_model.log_likelihood(Datum.data)
-    else:
-        approximate_likelihood = -approximate_model.log_likelihood()
-
-    assert_allclose(gpr_likelihood, approximate_likelihood, rtol=1e-6)
+    gpr_likelihood = gpr_model.log_marginal_likelihood()
+    approximate_likelihood = model_maximum_log_likelihood_objective(approximate_model, Datum.data)
```

Comment:
we know this is a GPR so we don't need to use this function

Reply:
this is `approximate_model` which can be SGPR, VGP, or SVGP